src/PrestaShopBundle/Controller/Admin/Sell/Catalog/FeatureController.php

```diff
@@ -53,101 +54,35 @@
 /**
  * Controller responsible for "Sell > Catalog > Attributes & Features > Features" page
  */
-class FeatureController extends FrameworkBundleAdminController implements LegacyControllerBridgeInterface, LegacyListControllerBridgeInterface
+class FeatureController extends FrameworkBundleAdminController implements FrameworkBridgeControllerInterface
```

Comment:
```suggestion
class FeatureController extends FrameworkBundleAdminController implements FrameworkBridgeControllerInterface, FrameworkBridgeListControllerInterface
```

Shouldn't it implement this interface? If not then the interface is maybe useless actually since you can just use the build method from the trait. It's not like the `FrameworkBridgeControllerInterface` which is required because we need a public method `getControllerConfiguration` that is called in the listener

Reply:
yes, as we discussed List interface is becoming useless (it used to require a method for building list config, getting table name, identifiers etc.. from controller, but as discussed it doesn't make much sense because there are some pages having multiple grids in one controller, so we better keep the trait to simplify adding as many grids as controller needs), I'll probably delete the interface after double-checking.

Reply:
Ok so indeed we should remove this now useless interface since it won't be used anymore and instead rely on the trait which provides a helper method to build the list.